products/fairphone.md

```diff
@@ -7,38 +7,44 @@ releasePolicyLink: https://forum.fairphone.com/t/fairphone-product-release-cycle
 releaseColumn: false
 releaseDateColumn: true
 discontinuedColumn: true
+customColumns:
+-   property: supportedAndroidVersions
+    position: after-release-column
+    label: Supported Android
+    description: Supported Android versions # https://support.fairphone.com/hc/en-us/articles/9979180437393-Fairphone-OS
+    link: https://endoflife.date/android
```

Comment:
```suggestion
    description: Supported Android version range
    link: https://support.fairphone.com/hc/en-us/articles/9979180437393-Fairphone-OS
```

I would find the Fairphone link more informative, WDYT ?

Reply:
depends...in this specific case I agree I prefer the Fairphone link which is very clear and informative.
But it's a link that'll be on the column Android, and this Android column will exist on several pages (Lineage, Fairphone, Pixel, OnePlus, etc.) so for the sake of continuity we could expect to always redirect to the Android OS page.
Looks good to me either way

Reply:
The primary goal of endoflife.date is to be useful and informative, so let's use the fairphone link over the endoflife.date one.